modules/ROOT/pages/building-https-proxy.adoc

```diff
@@ -3,62 +3,49 @@ ifndef::env-site,env-github[]
-Make sure that your TLS context's target is of type "Mule", and that the Secret Group is marked as "downloadable".
-+
-If you are not familiar with the secrets manager concepts, you should also review the xref:anypoint-security::asm-secret-group-concept.adoc[Secrets Group Basic Concepts].
+** API Manager permissions +
+To apply a TLS context to an API proxy in API Manager, you need the *Manage API Configuration* role.
+See the xref:access-management::permissions-by-product.adoc#api-manager[API Manager] section of the Permissions
+by Product page for more information.
+* xref:anypoint-security::asm-secret-group-creation-task.adoc[Created a Secret Group] +
+Enable the *Secret Group Downloadable* checkbox.
+* xref:anypoint-security::asm-secret-group-creation-task.adoc#add-a-tls-context[Added a TLS Context] +
+Set TLS context's target to *Mule*.
 
 == Configure your API
 
-. In API Manager, navigate to *Manage API* > *Manage API From Exchange*.
-. Configure your API proxy asset. 
-+
-For more information, see xref:api-manager::manage-exchange-api-task.adoc[Managing an API from Exchange]. 
-+
-While configuring your API Proxy, make sure you also select the following configurations:
-+
-.. Select *Endpoint with Proxy* in the *Managing Type* fields
-.. If you want to manage the API using Mule 4 or later (recommended), accept the default selection.
-+
-The parameters that you specify differ based on the Mule version you select. 
-. Click the *Add TLS Context* in the *TLS Context for Outbound traffic* field.
-+
-image::building-https-proxy-c7589.png[65%,85%]
-+
+include::partial$task-configure-proxy.adoc[tags=first-steps;target-type;mid-steps;asset-type-options;mid-steps2;raml-oas-version]
+
+. Click *Add TLS Context* for *TLS Context for Outbound traffic*.
 .. Select the secret group where you hosted your TLS Context from the *Secret Group* drop-down list.
 .. Select your TLS Context for your HTTPS API Proxy from the *TLS Context* drop-down.
 +
-image::building-https-proxy-7543c.png[35%,55%]
-+
-[TIP]
-If you can't see any context, make sure you have the right permissions mentioned in the prerequisites above.
-
-Additionally, if you want to enable SSL inbound traffic:
+[NOTE]
+If you can't see a context, check that you have the right permissions mentioned in the prerequisites above.
 
-. Click *Advanced Options*.
-. On the *Scheme* field, select *HTTPS*. +
+. Optionally, if you want to enable SSL for inbound traffic:
+.. Select *HTTPS* for the *Scheme*. +
 This makes the *TLS Context for Inbound Traffic* field appear.
-. Select *Add TLS Context*.
+.. Select *Add TLS Context*.
 .. Select the secret group where you hosted your TLS Context from the *Secret Group* drop-down.
 .. Select your TLS Context for your HTTPS API Proxy from the *TLS Context* drop-down.
-. Click *Save*
+include::partial$task-configure-proxy.adoc[tags=last-steps]
 
```

Comment:
I would change "This makes the *TLS Context for Inbound Traffic* field appear" to "The TLS Context for Inbound Traffic field appears."

Reply:
will fix